datafusion/src/physical_plan/mod.rs

```diff
@@ -343,7 +343,8 @@ pub enum Partitioning {
     RoundRobinBatch(usize),
     /// Allocate rows based on a hash of one of more expressions and the specified
     /// number of partitions
-    /// This partitioning scheme is not yet fully supported. See [ARROW-11011](https://issues.apache.org/jira/browse/ARROW-11011)
+    /// FIXME: This partitioning scheme is not yet fully supported.
```

Comment:
You can remove this comment - as the hash partitioning scheme is supported

Reply:
done